src/css/controls/imports/playback-label.less

```diff
@@ -15,17 +15,16 @@
 .jw-idle-label {
     border-radius: 50%;
     color: @white;
-    filter: drop-shadow(1px 1px 5px rgba(12, 26, 71, 0.25));
     font: normal 16px/1 Arial, Helvetica, sans-serif;
     position: relative;
-    transition: background-color @default-transition;
-    transition-property: background-color, filter;
     -webkit-font-smoothing: antialiased;
 }
 
 .jw-state-idle {
     .jw-icon-display {
         &.jw-idle-label {
+            filter: drop-shadow(1px 1px 5px rgba(12, 26, 71, 0.25));
```

Comment:
`filter` is needed for svgs, but for elements containing only text, we should be using `text-shadow` https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow